### ISSUE TITLE
Applied Energistics 2 - Infinite Power

### ISSUE THREAD

**Issue by jeremiahwinsley** [ReikaKalseki/Reika_Mods_Issues#58]
I reported [an issue](https://github.com/AppliedEnergistics/Applied-Energistics-2/issues/1384) with the Energy Acceptor producing infinite power. @thatsIch said it looks like a RotaryCraft bug, so I'm reporting it here.

**Comment by ReikaKalseki** [comment]
This is almost certainly on AE's end.

**Comment by thatsIch** [comment]
We changed nothing on our end in that compartment since v5, so any change you introduced to v6 most likely changed interaction. I tried it in a creative world and there were no events fired by using an industrial coil (placed in creative). I updated the integration with RtC to v5 at that time and I am pretty sure, that my setup worked. 

So as long you do not try it for yourself in an actual world, I will stick with that.
